### PR TITLE
Locks the research paper only to the hermit

### PR DESCRIPTION
**Pull request by StrangeWeirdKitten**
## About The Pull Request

Removes the research paper from everywhere and locks it only to the hermit.

## Why It's Good For The Game

This item was only specifically meant to go to the hermit. I defended it at first, but people are correct that prisoners shouldn't be building entire robotic factories and I think the age of easy alien tools in science should probally pass.

## Proof Of Testing

image

image

## Changelog

:cl:
balance: The research paper crafting recipe has been locked to the hermit.
/:cl:

### CODE REVIEW

**Comment by xXPawnStarrXx** [comment]
It's a shame we can't have fun things without other's ruining it. It was a good run while it lasted.

File: modular_zubbers/code/modules/mob_spawn/ghost_roles/mining_roles.dm [hunk 1]
```diff
@@ -20,3 +20,13 @@
...
+
+/datum/outfit/hermit
+	backpack_contents = list(/obj/item/research_paper = 1)
+
+/datum/outfit/hermit/post_equip(mob/living/carbon/human/hermit, visualsOnly)
+	. = ..()
+	if(visualsOnly)
+		return
+	hermit.mind?.teach_crafting_recipe(/datum/crafting_recipe/research_paper)
+	to_chat(hermit, span_notice("You learn the recipe of the <b>research paper</b> to craft everything from nothing."))
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/mob_spawn/ghost_roles/mining_roles.dm:32]:
```suggestion
	to_chat(hermit, span_notice("You learn the recipe for the <b>research paper</b>, giving you the ability to craft everything from nothing."))
```

**Review by Majkl-J — CHANGES_REQUESTED** [review]

**Review by Majkl-J — APPROVED** [review]
This is the first time I downright kill something with no potential path of redemption.

**Comment by MajManatee** [comment]
born to have fun, forced to do xenoarch (the most boring thing possible)

**Comment by MothionTechnologies** [comment]
I think it might be better to just remove it, since so few people even play hermit and even those that play usually just go find the station instead of doing survival.

**Pull request merged**